### PR TITLE
Accessibility improvements for OAuth Consent view

### PR DESCRIPTION
**Pull request by sherizada-okta**
## Description:
Accessibility improvements for OAuth Consent and AdminConsent required views:
- add focus state to input elements inside auth container
- add aria attributes for logo images
- match button tab order to visual hierarchy, i.e. `Don't allow` is focused via tab before `Allow`

Improvements for all of SIW:
- update font color variables `$header-text-color`, `$medium-text-color`, and `$text-color-light`
- add a main landmark to SIW 
- add a skip link to SIW

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Demo with screen reader: https://okta.box.com/s/7a4pkpk7596zs9ghpymgo2ptrm1dqctu

Skip link shows only on focus:
Screen Shot 2020-11-20 at 12 19 33 PM

Consent
Screen Shot 2020-11-19 at 2 00 15 PM
Screen Shot 2020-11-19 at 2 00 21 PM

AdminConsent
Screen Shot 2020-11-12 at 11 22 07 AM
Screen Shot 2020-11-12 at 11 22 00 AM

### Issue:

- [OKTA-334550](https://oktainc.atlassian.net/browse/OKTA-334550)
- [OKTA-334664](https://oktainc.atlassian.net/browse/OKTA-334664)
- [OKTA-337231](https://oktainc.atlassian.net/browse/OKTA-337231)

### CODE REVIEW

**Review by haishengwu-okta — COMMENTED** [review]
- Are those color from Odyssey?
- Change color variables meaning impact ALL widget pages. Did you double check we're good on this?

**Comment by pradeepdewda-okta** [comment]
> * Are those color from Odyssey?
> * Change color variables meaning impact ALL widget pages. Did you double check we're good on this?

Yeah, it comes from product team itself. They probably need these change for all pages.

**Review by ZoeMonosson-okta — APPROVED** [review]
This looks good to me!

**Comment by ZoeMonosson-okta** [comment]
> * Are those color from Odyssey?
> * Change color variables meaning impact ALL widget pages. Did you double check we're good on this?

@haishengwu-okta yes, the colors are pulled from Odyssey. #1d1d21 is the same as Gray900, used for titles and body copy. #6e6e78 is the same as gray600, which is used for caption text.

**Comment by haishengwu-okta** [comment]
Would be nice to add comment to those new color how it corresponds to odyssey. (e.g. `Gray900`)

**Comment by sherizada-okta** [comment]
@haishengwu-okta this is ready for a review again, thanks

**Comment by sherizada-okta** [comment]
@ZoeMonosson-okta please see the updated demo videos in the PR description of the Consent view, the ones I sent you were of AdminConsent which will also have the same updates. The difference is that Consent has a link in the header which comes before the buttons when tabbing. I will add the skip link in another PR when styling is decided.

**Comment by ZoeMonosson-okta** [comment]
LGTM!

**Review by pradeepdewda-okta — APPROVED** [review]

File: src/views/shared/AuthContainer.js [hunk 1]
```diff
@@ -14,9 +14,10 @@ import { View } from 'okta';
 import Enums from 'util/Enums';
 const CAN_REMOVE_BEACON_CLS = 'can-remove-beacon';
 export default View.extend({
+  tagName: 'main',
```
Comment (haishengwu-okta) [inline src/views/shared/AuthContainer.js:17]: https://caniuse.com/?search=main
No IE 11 support.. 🤦

Comment (sherizada-okta) [reply src/views/shared/AuthContainer.js:17]: Solution: https://stackoverflow.com/questions/35820429/main-element-not-working-in-internet-explorer-11#:~:text=1%20Answer&text=The%20HTML5%20main%20element%20is,element%20for%20width%20to%20work.

**Pull request closed without merging**